site/include/form.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+
+$FormErrors = array();
+
+function TextBox($FieldName){
+	global $FormErrors;
+	echo"<input type='text' value='".@$_REQUEST[$FieldName]."'
+	name='$FieldName' />";
+	echo"$FormErrors[$FieldName]";
+}
+
+function EditTextBox($Value, $FieldName){
+	global $FormErrors;
+	echo"<input type='text' value='$Value' name='$FieldName' />";
+	echo"$FormErrors[$FieldName]";
+}
+
+function ReadOnlyTextBox($Value, $FieldName){
+	global $FormErrors;
+	echo"<input type='text' value='$Value' name='$FieldName' readonly />";
+	echo"$FormErrors[$FieldName]";
+}
+
+function DropDown($FieldName, $Option1, $Option2, $Option3, $Option4){
+	global $FormErrors;
+	echo"<select value='".@$_REQUEST[$FieldName]."' name='$FieldName'>
+	<option value='$Option1'>
```

Comment:
I'm only mentioning this randomly, but this is another place where I'd indent everything between the opening and closing lines of the echo. This all blends together as one big mess of code instead of looking like an echo with some markup inside